src/util/jszip.js

```diff
@@ -1,9155 +1,9412 @@
 /*!
```

Comment:
Thanks, Miguel. I thought proj4-src already has the pure AMD interface. Did we not handle that in a previous pull request?